### PR TITLE
output skip deployment message when NONE deploymentType is set in pom.xml

### PR DESCRIPTION
**Pull request by SummerSun**

### CODE REVIEW

File: azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/artifacthandler/FTPArtifactHandlerImpl.java [hunk 1]
```diff
@@ -45,6 +45,8 @@ protected boolean isResourcesPreparationRequired(final DeployTarget target) {
 
     @Override
     public void publish(final DeployTarget target) throws IOException, MojoExecutionException {
+        this.mojo.getLog().info(String.format(DEPLOY_START, target.getType(), target.getName()));
```
Comment (jdneo) [inline azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/artifacthandler/FTPArtifactHandlerImpl.java]: `Deploying Web App to xxx`? this log looks strange.

File: azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/artifacthandler/ArtifactHandler.java [hunk 2]
```diff
@@ -10,4 +10,7 @@
 
 public interface ArtifactHandler<T extends DeployTarget> {
     void publish(T deployTarget) throws Exception;
+
+    String DEPLOY_START = "Deploying '%s' to '%s'...";
```
Comment (jdneo) [inline azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/artifacthandler/ArtifactHandler.java]: How about change the sequence to:

```
Trying to deploy artifact to xxx...

Detect <deploymentType> is set to NONE, skip deployment.

Deployment finished.

```

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java [hunk 3]
```diff
@@ -101,11 +101,11 @@ protected void deployArtifacts() throws Exception {
                 target = new WebAppDeployTarget(app);
             }
 
-            getLog().info(String.format(DEPLOY_START, target.getType(), target.getName()));
+            getLog().info(DEPLOY_START);
```
Comment (jdneo) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java]: How about change the sequence to:

```
Trying to deploy artifact to xxx...

Detect <deploymentType> is set to NONE, skip deployment.

Deployment finished.

```

File: azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/artifacthandler/FTPArtifactHandlerImpl.java (hunk 1 above)
Comment (SummerSun) [reply azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/artifacthandler/FTPArtifactHandlerImpl.java]: Typo, should be Deploying to Web App xxx, to be different from Deploying to Deployment Slot xxx.

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java (hunk 3 above)
Comment (SummerSun) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java]: Looks good.

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/NONEArtifactHandlerImpl.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+public class NONEArtifactHandlerImpl implements ArtifactHandler {
+    private AbstractWebAppMojo mojo;
+
+    public NONEArtifactHandlerImpl(final AbstractWebAppMojo mojo) {
+        this.mojo = mojo;
+    }
+
+    @Override
+    public void publish(DeployTarget deployTarget) {
+        this.mojo.getLog().info("Detect <deploymentType> was set to NONE, skip deployment.");
```
Comment (jdneo) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/NONEArtifactHandlerImpl.java]: was -> is

**Pull request merged**